app/assets/themes/fearlesscities/_body.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+
+body {
+  font-family: 'Lora', 'Poppins', sans-serif;
```

Comment:
Make sure all `font-family` rules have one of these two values:
`font-family: "Lora", serif;`
`font-family: "Poppins Bold", sans-serif;`